novawallet/Modules/Swaps/Base/View/SwapNetworkFeeViewCell.swift

```diff
@@ -1,11 +1,11 @@
 import SoraUI
 
-final class SwapNetworkFeeViewCell: RowView<SwapNetworkFeeView>, StackTableViewCellProtocol {
+final class SwapNetworkFeeViewCell: RowView<NetworkFeeInfoView>, StackTableViewCellProtocol {
```

Comment:
What is the purpose to change the viewModel?

Reply:
I just renamed class SwapNetworkFeeView  because now it is no longer exclusively used in Swaps; it is also using in Staking.